main.css

```diff
@@ -123,6 +131,28 @@ h3.headline {
 }
 
 
+/* Banner ---------------------------------------- */
+
+.banner {
+  padding: 1em;
+  width: 360px;
+  font-size: 1.5rem;
+}
+
+@media screen and (min-width: 60em) {
+  .banner {
+    margin: 0 auto 30px;
+    padding: 0 1em;
+    width: 520px;
+    border-right: 2px solid hsl(48, 89%, 60%);
+    border-left: 2px solid hsl(48, 89%, 60%);
+    border-radius: 5px;
```

Comment:
This is a very interesting effect. I thought at first you used pseudo elements. :clap: